community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/TokenNameLookup.java

```diff
@@ -42,4 +46,26 @@
      * @return name of property key token with given id.
      */
     String propertyKeyGetName( int propertyKeyId );
+
+    default String[] entityTokensGetNames( EntityType type, int[] entityTokenIds )
+    {
+        IntFunction<String> mapper;
+        if ( type == EntityType.NODE )
+        {
+            mapper = this::labelGetName;
+        }
+        else if ( type == EntityType.RELATIONSHIP )
+        {
+            mapper = this::relationshipTypeGetName;
+        }
+        else
+        {
+            return new String[0];
```

Comment:
Why not throwing exception? This seems like a weird return value. From such mapping functions it is quite natural to expect result array of the same size as the input.